apps/fyler/src/fyler_sup.erl

```diff
@@ -9,7 +9,7 @@
 %% Supervisor callbacks
 -export([init/1]).
 
--export([start_worker/1, stop_worker/1, start_category_manager/1]).
+-export([start_worker/1, stop_worker/1, start_category_manager/1, start_callback_worker/2]).
```

Comment:
Line 12 is too long: -export([start_worker/1, stop_worker/1, start_category_manager/1, start_callback_worker/2])..